aws_lambda_powertools/utilities/data_classes/__init__.py

```diff
@@ -16,7 +16,7 @@
 from .kinesis_firehose_event import KinesisFirehoseEvent
 from .kinesis_stream_event import KinesisStreamEvent
 from .lambda_function_url_event import LambdaFunctionUrlEvent
-from .s3_event import S3Event
+from .s3_event import S3Event, S3EventBridgeEvent
```

Comment:
Let's break this into a separate PR. That way we have two explicit PRs we can look back on those two features for distinct utilities.

Reply:
Will do, probably over the weekend though.

Reply:
This PR now only contains changes related to the parser utility. Dataclasses event will be a separate PR, as you suggested.

Reply:
Created PR for dataclasses #1990